omnigibson/utils/urdfpy_utils.py

```diff
@@ -0,0 +1,4091 @@
+import copy
```

Comment:
is this file copied verbatim from another library? Should we cite them properly?

Reply:
done!